MdeModulePkg/Bus/Pci/PciHostBridgeDxe/PciRootBridgeIo.c

```diff
@@ -1347,6 +1355,16 @@ RootBridgeIoMap (
 
   RootBridge = ROOT_BRIDGE_FROM_THIS (This);
 
+  // MU_CHANGE [BEGIN] - Allow platform to disallow unprotected DMA access when done before IOMMU install.
+  if (FeaturePcdGet (PcdRequireIommu) && (mIoMmu == NULL)) {
+    DEBUG ((DEBUG_ERROR, "%a - mIoMmuProtocol is NULL!\n", __FUNCTION__));

```

Comment:
If you could, please do just a quick replace of `__FUNCTION__` with `__func__` in the modifications.

Reply:
Updated.